### ISSUE TITLE
Markdown tests fail on windows due to line ending different from unix one

### ISSUE THREAD

**Issue by luigi-asprino** [SPARQL-Anything/sparql.anything#309]
This relates to #266 
DOS line ending is `\r\n` while Unix is `\n`

https://www.cs.toronto.edu/~krueger/csc209h/tut/line-endings.html#:~:text=Text%20files%20created%20on%20DOS,(%22%5Cn%22).

Therefore, the expected graph contains `\r\n` as line-ending and the graph generated on windows has just `\n`.

**Comment by luigi-asprino** [comment]
On my mac (and on the remote repository I suppose) the input files of the tests contain only line feed characters (`\n`).
But, after pulling the files on windows, the line ending becomes `\r\n`.
Therefore, I thought that there must be something in git to control how the line endings are handled.
I found that this can be controlled via .gitattributes file https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings
With [f555808](https://github.com/SPARQL-Anything/sparql.anything/commit/f55580839b10aef60fe9e1aea0811fe52eb7507c) I added a .gitattributes file which forces (I hope) the files of the repository to be pulled with `\n` as line ending character.
